test/test_playbooks/tasks/activation_key_copy.yml

```diff
@@ -21,7 +22,7 @@
     content_view: "{{  activation_key_content_view }}"
     subscriptions: "{{ activation_key_subscriptions }}"
     auto_attach: "{{ activation_key_auto_attach }}"
-    state: 'copied'
+    state: "{{ activation_key_state }}"
```

Comment:
This file seems unneeded.

Reply:
Yeah I got that from the diff posted to #225. Should I revert that change and leave it as it is?

Reply:
Oh, i was too fast. That file _is_ needed. But i am not quite sure how it is meant to be used. In doubt leave it as is (not part of your pr...).

Reply:
I meant, not adding it to your patch, and leaving the file as in master.